hermeto/core/models/sbom.py

```diff
@@ -130,11 +130,9 @@ class Metadata(pydantic.BaseModel):
     tools: list[Tool] = [Tool(vendor="red hat", name=f"{APP_NAME}")]
 
 
-def spdx_now() -> str:
-    """Return a time stamp in SPDX-compliant format.
-
-    See https://spdx.github.io/spdx-spec/v2.3/search.html?q=date
-    for details.
+def timestamp_now() -> str:
```

Comment:
Hold your horses here - if you want to get a timestamp, you don't need a dedicated function for it. The sheer need for `spdx_now` stems from the fact that we needed an **explicit** string format to comply with the SPDX spec. Therefore, if you need a timestamp outside of SPDX's context, just use `datetime.datetime.now` directly. For me, keeping `spdx_now` is a very clear indicator that we need to something slightly differently compared to just calling `datetime` with its default settings and be done with it.